internal/automation/automation_test.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package automation
+
+import "testing"
+
+func TestAutomationCmdRun(t *testing.T) {
+	cmd := newAutomationCommand()
+	if err := cmd.Run(t.Context(), []string{"version"}); err != nil {
```

Comment:
test the other commands also

Reply:
The other commands are not easy to test since they are running cloud build triggers.

The commands all reuse the exiting logic in `runCommandFn` and I'll refactor further once all the commands are refactored. This PR only changes surface, how to parse the flags, for example.

I've added a [unit test](https://github.com/googleapis/librarian/blob/8047f1cff854cc02354a3c53d386845543c5cacc/internal/automation/generate_test.go#L56) to verify the flags are correctly passed to `runCommandFn`.